src/App.tsx

```diff
@@ -13,46 +13,81 @@ import AddPage from "./pages/AddPage";
 import NewTodoPage from "./pages/NewTodoPage";
 import ItemDetail from "./pages/itemDetail";
 
+export enum Frequency {
```

Comment:
enums are used when you want to only allow a certain choice of options:

https://www.typescriptlang.org/docs/handbook/enums.html